addOns/retire/src/main/java/org/zaproxy/addon/retire/model/Repo.java

```diff
@@ -307,4 +309,58 @@ private static boolean isGoodCandidate(String version) {
         }
         return isAllZeros != 0; // Not a good value if all zero
     }
+
+    static int getRiskForSeverity(String severity) {
+        if (severity == null || severity.isEmpty()) {
+            severity = "medium";
+        }
+        severity = severity.toLowerCase(Locale.ROOT);
+        int risk = -1;
+        risk = SEVERITY_MAP.get(severity) == null ? Alert.RISK_MEDIUM : SEVERITY_MAP.get(severity);
+        return risk;
+    }
+
+    public static class VulnerabilityData {
+        public static final VulnerabilityData EMPTY = new VulnerabilityData(true);
+        private Set<String> cves = new HashSet<>();
+        private Set<String> info = new HashSet<>();
```

Comment:
It could initialise in the constructors with `EMPTY` using immutable sets, this would also ensure that `EMPTY` does not get accidentally modified and end up leaking data.